Core/src/org/sleuthkit/autopsy/modules/ileappanalyzer/ILeappAnalyzerIngestModule.java

```diff
@@ -0,0 +1,263 @@
+                try {
+                    Files.createDirectories(moduleOutputPath);
+                } catch (IOException ex) {
+                    logger.log(Level.SEVERE, String.format("Error creating iLeapp output directory %s", moduleOutputPath.toString()), ex);
+                    return ProcessResult.ERROR;
+                }
+
+                statusHelper.progress(NbBundle.getMessage(this.getClass(), "ILeappAnalyzerIngestModule.processing.file", iLeappFile.getName()), filesProcessedCount);
+                ProcessBuilder iLeappCommand = buildiLeappCommand(moduleOutputPath, iLeappFile.getLocalAbsPath(), iLeappFile.getNameExtension());
+                try {
+                    int result = ExecUtil.execute(iLeappCommand, new DataSourceIngestModuleProcessTerminator(context));
+                    if (result != 0) {
+                        logger.log(Level.SEVERE, String.format("Error running iLeapp, error code returned %d", result)); //NON-NLS
+                        return ProcessResult.ERROR;
+                    }
+                    
+                    addILeappReportToReports(moduleOutputPath, currentCase);
+
+                } catch (IOException ex) {
+                    logger.log(Level.SEVERE, String.format("Error when trying to execute iLeapp program against file %s", iLeappFile.getLocalAbsPath()), ex);
+                    return ProcessResult.ERROR;
+                }
+
+                if (context.dataSourceIngestIsCancelled()) {
+                    logger.log(Level.INFO, "ILeapp Analyser ingest module run was canceled"); //NON-NLS
+                    return ProcessResult.OK;
+                }
+
+                ProcessResult fileProcessorResult = iLeappFileProcessor.processFiles(dataSource, moduleOutputPath, iLeappFile);
+
+                if (fileProcessorResult == ProcessResult.ERROR) {
+                    return ProcessResult.ERROR;
+                }
+                
+                filesProcessedCount++;
+            }
+
+        }
+
+        IngestMessage message = IngestMessage.createMessage(IngestMessage.MessageType.DATA,
+                Bundle.ILeappAnalyzerIngestModule_has_run(),
+                Bundle.ILeappAnalyzerIngestModule_completed());
+        IngestServices.getInstance().postMessage(message);
+        return ProcessResult.OK;
+    }
+
+    /**
+     * Find the files to process that will be processed by the iLeapp program
+     *
+     * @param dataSource
+     *
+     * @return List of abstract files to process.
+     */
+    private List<AbstractFile> findiLeappFilesToProcess(Content dataSource) {
+
+        List<AbstractFile> iLeappFiles = new ArrayList<>();
+
+        FileManager fileManager = getCurrentCase().getServices().getFileManager();
+
+        // findFiles use the SQL wildcard # in the file name
+        try {
+            iLeappFiles = fileManager.findFiles(dataSource, "%", "/"); //NON-NLS
+        } catch (TskCoreException ex) {
+            //Change this
```

Comment:
I think you can remove this comment, assuming you did change it

Reply:
Comment removed